addons/website/static/src/snippets/s_dynamic_snippet/options.js

```diff
@@ -224,10 +215,8 @@ const dynamicSnippetOptions = options.Class.extend({
             if (!this.dynamicFilterTemplates[selectedTemplateId]) {
                 this.$target.get(0).dataset['templateKey'] = dynamicFilterTemplates[0].key;
                 this.isOptionDefault['templateKey'] = true;
-                setTimeout(() => {
-                    this._templateUpdated(dynamicFilterTemplates[0].key, selectedTemplateId);
-                    this._refreshPublicWidgets();
-                });
+                this._templateUpdated(dynamicFilterTemplates[0].key, selectedTemplateId);
+                this._refreshPublicWidgets();
```

Comment:
`[2]` being a fix already applied on 15.0, shouldn't removing the `setTimeout` also be done on 15.0 ?